### DISCUSSION TITLE
A mechanism to name claims

### DISCUSSION THREAD

**Discussion by Noxillio** [GriefPrevention/GriefPrevention#1846] [Ideas]
Add a command that lets players name their claims. For example, you would have to stand in the claim and do `/claimname|cn <name>`.

**Comment by Jikoo** [comment]
[ClaimClassifier](https://github.com/GPAddons/ClaimClassifier) exists for this.

**Reply by xSepticSidx** [reply]
Is ClaimClassifiercompatable with current versions?

**Reply by Jikoo** [reply]
Presumably. GP hasn't made breaking changes for years.
